releasenotes/notes/vqe-factory-improvements-02f411513f11907c.yaml

```
@@ -0,0 +1,9 @@
+---
+issues:
```

```suggestion
deprecations:
```
This should be under a `deprecations` section - `issues` section is for known bugs i.e. issues that are present in the release that we want to make users aware of that we know are there.

---

Good to know.